app/unitPages.rb

```diff
@@ -419,15 +420,21 @@ def getIssues(unit_id)
 # Landing page data does not pass arguments volume/issue. It just gets most recent journal
 def getJournalIssueData(unit, unit_attrs, volume=nil, issue=nil)
   display = unit_attrs['magazine_layout'] ? 'magazine' : 'simple'
-  issues = getIssues(unit.id)
+  # Returns array of issue IDs
+  publishedIssues = Issue.distinct.select(Sequel[:issues][:id]).
+    join(:sections, issue_id: :id).
+    join(:items, section: Sequel[:sections][:id]).
+    filter(Sequel[:issues][:unit_id] => unit_id,
```

Comment:
I'm getting `undefined local variable or method 'unit_id'` on line 427 when trying to access  http://localhost:4001/uc/alephucla/0/0 (with either the dev or prd database).